zeq/subscriber.h

```diff
@@ -19,6 +19,16 @@ namespace detail { class Subscriber; }
 /**
  * Subscribes to Publisher to receive events.
  *
+ * If the subscriber is in the same session as discovered publishers, it
+ * automatically subscribes to those publishers. Publishers from the same
+ * application instance are not considered though.
```

Comment:
Is there a way to connect publishers and subscribers from the same application without having to use explicit URIs?